src/js/components/Button/StyledButtonKind.js

```diff
@@ -106,6 +111,10 @@ const kindPartStyles = (obj, theme, colorValue) => {
           theme,
         )};
       `);
+    if (obj.border.radius)
+      styles.push(css`
+        border-radius: ${obj.border.radius};
+      `);
```

Comment:
If we are already providing `border-radius` styling from `basicStyles`, why is this needed in addition? we might end up with two css values of `border-radius`?

Reply:
We often have two CSS rules, last one wins. This allows each kind to have a different radius.